packages/server-wallet/src/protocols/__test__/direct-funding.test.ts

```diff
@@ -4,7 +4,7 @@ import {some, none} from 'fp-ts/lib/Option';
 
 import {protocol} from '../direct-funding';
 import {alice} from '../../wallet/__test__/fixtures/signingWallets';
-import {Address, Uint256} from '../../type-aliases';
+import {Uint256} from '../../type-aliases';
```

Comment:
Hmm, I think this should use the `Uint256` from `wallet-core`. Casting to that seems fine in tests, instead of `BN.from(5)`, but we want to make sure that things that are a `Uint256` have passed some kind of a data validator, which is the point of the "branded" `Uint256` type